### PR TITLE
Add a PropertyKey on whether FileSystemMaster shall update accesstime

### LINKED ISSUE #13495: Add a switch about whether FileMaster update accessTime when reading files
**Describe the solution you'd like**
Add a field in alluxio-site.properties，if the field not set, the master will update accessTime when reading file.

**Urgency**
The master might under great pressure in high concurrency if update accsstime when reading file.

### PR DESCRIPTION
**Pull request by kuszz**
Fix #13495

### CODE REVIEW

**Comment by kuszz** [comment]
Add "alluxio.master.read.accesstime.update=false" in conf/alluxio-site.properties will let filemaster don't update accesstime when reading files, and if don't add, the master will performs like before

**Comment by apc999** [comment]
@jiacheliu3 @ggezer , this is a community contribution to improve master scalability on high concurrency. Can you help review? If more info is needed, feel free to ask more questions

**Comment by jiacheliu3** [comment]
@apc999 @ggezer  A high level question: which do you guys like better:
A: a switch ON/OFF for file access time update
B: specify a granularity like 1min, if the file last access time is 1min, then we don't update.

I feel like option B might help more people where it's hard to totally disable access time but can allow losing precision.

**Comment by maobaolong** [comment]
Hi @jiacheliu3 , do you mean we can almost switch off the access time update by set  `alluxio.master.file.access.time.update.precision` to a large value? Let's say  `36500d`.

**Comment by jiacheliu3** [comment]
> Hi @jiacheliu3 , do you mean we can almost switch off the access time update by set `alluxio.master.file.access.time.update.precision` to a large value? Let's say `36500d`.

Oh actually I didn't know this property. I don't think [this change](https://github.com/Alluxio/alluxio/commit/b7a3878f6021518e70c5ebbd9dee4a681bbd33e9) does 100% what you need (because when inode metadata is updated, the access time is updated synchronously), but I guess that's worth a try.

And if you want to change how access time is updated, seems `AccessTimeUpdater` is the place to go to.

File: core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java [hunk 1]
```diff
@@ -226,6 +228,9 @@
     implements FileSystemMaster, DelegatingJournaled {
   private static final Logger LOG = LoggerFactory.getLogger(DefaultFileSystemMaster.class);
   private static final Set<Class<? extends Server>> DEPS = ImmutableSet.of(BlockMaster.class);
+  private static final boolean READ_ACCESSTIME_UPDATE =
+      new InstancedConfiguration(ConfigurationUtils.defaults())
+      .getBoolean(PropertyKey.MASTER_READ_ACCESSTIME_UPDATE);
```
Comment (apc999) [inline core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java:233]: use `ServerConfiguration.global()`

**Comment by ggezer** [comment]
Hi @kuszz, Let's park this PR until we complete the discussion in the ticket (https://github.com/Alluxio/alluxio/issues/13495) and understand the issue better.

**Comment by kuszz** [comment]
> Hi @kuszz, Let's park this PR until we complete the discussion in the ticket (#13495) and understand the issue better.

It seems no need to add new switch, so close this pr, sorry

**Pull request closed without merging**